internal/controlplane/handlers_authz.go

```diff
@@ -530,32 +495,70 @@ func (s *Server) removeInvite(
 	role authz.Role,
 	email string,
 ) (*minder.RemoveRoleResponse, error) {
-	prj := targetPrj.String()
-	// Get all invitations for this email, project and role
-	inviteToRemove, err := s.store.GetInvitationByEmailAndProjectAndRole(ctx, db.GetInvitationByEmailAndProjectAndRoleParams{
+	// Get all invitations for this email and project
+	invitesToRemove, err := s.store.GetInvitationsByEmailAndProject(ctx, db.GetInvitationsByEmailAndProjectParams{
```

Comment:
Would it be possible to have an SQL query to find an invite by role, e-mail and project (or extend GetInvitationsByEmailAndProject to accept an optional role)? Then you wouldn't have to do the filtering yourself and could just check if you got 0 replies, 1 or more than 1 which would be an internal error.

Reply:
That was literally the line I removed :D

Reply:
now you know that I add comments in a stream without first reading the whole PR ;-)